README.md

```diff
@@ -4,17 +4,35 @@ Allows zero-downtime deployments of applications within Cloud Foundry, with no a
 
 ## Usage
 
-Requires [the `cf` CLI](https://docs.cloudfoundry.org/devguide/installcf/install-go-cli.html) v6.12.0+ (for [this fix](https://www.pivotaltracker.com/n/projects/892938/stories/96041780)).
-
-1. Run `npm install -g cf-blue-green`.
-    * Requires Node.js, just for distribution via NPM. If you don't want to install it, simply:
-        1. Download [the script](bin/cf-blue-green).
-        1. Run `chmod a+x cf-blue-green`.
-        1. Move the file somewhere in your PATH.
+1. Install [the `cf` CLI](https://docs.cloudfoundry.org/devguide/installcf/install-go-cli.html) < v6.12.0.
+    * See [Notes](#cf-cli) below for more info about the version restriction.
+1. Run `npm install -g cf-blue-green`
+    * See [Notes](#manual-installation) below for non-Node installation.
 1. Run `cf-blue-green <appname>` (instead of `cf-push`) from your application directory to deploy.
 
 This creates a copy of your already-running application, and safely switches traffic over to it. It's recommended that you try this script on a non-production application environment first, just to ensure that everything is switched over properly.
 
+## Notes
+
+### CF CLI
+
+v6.12.0+ has [an unresolved bug](https://www.pivotaltracker.com/n/projects/892938/stories/100594158) (as of this writing) that breaks `cf-blue-green`, so you will need an older version. To downgrade to 6.11.3 on Mac, if you use [Homebrew](http://brew.sh/):
+
+```bash
+brew uninstall cloudfoundry-cli
+brew install https://raw.githubusercontent.com/pivotal/homebrew-tap/cefcf8f7766b12d2945e93df53ce3aa3f00b7e5c/cloudfoundry-cli.rb
+```
+
+Note, however, that older versions have a _different_ issue, where [the `buildpack` parameter isn't respected](https://www.pivotaltracker.com/n/projects/892938/stories/96041780). In other words, don't use `cf-blue-green` (for now) if you use a custom buildpack. Sorry!
```

Comment:
I have it working with a custom buildpack with `cf version 6.12.1-56792aa-2015-07-06T22:48:09+00:00` I installed it with https://github.com/cloudfoundry/cli/releases/tag/v6.12.1